cilium/cmd/helpers.go

```diff
@@ -344,15 +344,15 @@ func updatePolicyKey(pa *PolicyUpdateArgs, add bool) {
 				err       error
 			)
 			if pa.isDeny {
-				err = policyMap.Deny(pa.label, pa.port, u8p, pa.trafficDirection)
+				err = policyMap.Deny(0, pa.label, pa.port, u8p, pa.trafficDirection)
 			} else {
-				err = policyMap.Allow(pa.label, pa.port, u8p, pa.trafficDirection, authType, proxyPort)
+				err = policyMap.Allow(0, pa.label, pa.port, u8p, pa.trafficDirection, authType, proxyPort)
```

Comment:
This was a half-baked attempt to model an API where the prefix length is exposed to allow port ranges to be supported later. Removed this so that this PR is much cleaner!